### ISSUE TITLE
Include the test folder on the release version

### ISSUE THREAD

**Issue by adbeda** [madmaze/pytesseract#208]
I'm attempting to bring `pytesseract` to a Conda-forge channel, so that conda enthusiast can use the package and part of the process of packing up a package involves testing the scripts. I can see that you have test files on github but [gun-zipped source code](https://github.com/madmaze/pytesseract/archive/v0.2.7.tar.gz) file on the **Release** tab doesn't come along with the test folder or the [test_pytessseract.py](https://github.com/madmaze/pytesseract/blob/master/tests/test_pytesseract.py) file.

Is it possible to include them on the release version of the source code so that the one we can make available on Conda can be well tested? For now, the only test I have is for the package to be able imported in conda environment.

**Comment by int3l** [comment]
Hi @adbeda 
We axed the images from the release when we started adding CI and unit testing ( #193 ).
I am not very familiar with conda-forge project, but Isn't there a better way of testing,
than adding the development assets to the package?

**Comment by adbeda** [comment]
Hi @int3l thanks for getting in touch. I'm sure you're familiar with conda software(package) manager released by the company [Anaconda](https://www.anaconda.com/). [The Conda-forge community](https://github.com/conda-forge) is a github organization that test and build software in a channel the can serve everyone.  Its free and open to all. 

The answer to your question is that yes, there are ways to test the package as is but the most common workflow I've came across involves testing with the test files shipped along with the package. Its not necessary to test everything and the test I've created checks if the software to downloads properly. Thats enough for now as it is the case for many other packages.
